### PR TITLE
Change account setup layout

### PR DESCRIPTION
**Pull request by wmontwe**
This changes the layout of the setup screens to use `Scaffold` and `TopAppBar`. It introduces a new `ResponsiveWidthContainer` that allows to limit a composable width based on the available screen width to better position it's content on large screens.

### CODE REVIEW

File: feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/common/AccountSetupTopAppBar.kt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+@Composable
+internal fun AccountSetupTopAppBar(
+    title: String,
+) {
+    TopAppBar(
+        title = title,
+        subtitle = stringResource(id = string.account_setup_top_bar_subtitle),
+        titleContentPadding = PaddingValues(
+            start = MainTheme.spacings.double,
```
Comment (cketti) [inline feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/common/AccountSetupTopAppBar.kt:21]: When there's no navigation icon in the toolbar, the title should start 16dp from the edge of the screen. It appears `MaterialTopAppBar` is automatically doing the right thing. Specifying `titleContentPadding` adds an additional 16dp, shifting the title/subtitle too far.

File: feature/account/setup/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,7 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="account_setup_title">K-9 Mail</string>
+    <string name="account_setup_top_bar_subtitle">K-9 Mail</string>
```
Comment (cketti) [inline feature/account/setup/src/main/res/values/strings.xml:3]: Things to remember for later: App-specific strings should live in a separate file.

**Review by cketti — APPROVED** [review]

File: feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/common/AccountSetupTopAppBar.kt (hunk 1 above)
Comment (wmontwe) [reply feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/common/AccountSetupTopAppBar.kt:21]: This is according to design to align content and title. Could be a topic to the next design iteration.

File: core/ui/compose/designsystem/src/main/kotlin/app/k9mail/core/ui/compose/designsystem/template/ResponsiveWidthContainer.kt [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+ * function to determine the screen size, and then applies the appropriate modifier.
+ *
+ * @param modifier Any modifier that should be applied to the outer container. This can be used
+ * to add padding, background colors, click events, etc.
+ * @param content The content to be placed inside this container. The content is expected to be
+ * a composable function.
+ *
+ * Example usage:
+ * ```
+ * AdaptiveLayoutWrapper {
```
[the remark this answers is not in the document]
Comment (wmontwe) [reply core/ui/compose/designsystem/src/main/kotlin/app/k9mail/core/ui/compose/designsystem/template/ResponsiveWidthContainer.kt:37]: Hm, that wasn't picked up when renaming -> fixed

File: feature/account/setup/src/main/res/values/strings.xml (hunk 2 above)
Comment (wmontwe) [reply feature/account/setup/src/main/res/values/strings.xml:3]: Right, so far they live in legacy and are not accessible here, maybe lift some of the strings into a core resource module.

**Pull request merged**